README.md

```diff
@@ -1 +1,69 @@
-# GatorGrade
+# GatorGrade: A Python Tool to Implement GatorGrader
+
+GatorGrade is a Python tool that executes GatorGrader, an automatic grading tool
+that can be used to check assignments through user-created checks. GatorGrade is
+the newer Python-based version of
+[GatorGradle](https://github.com/GatorEducator/gatorgradle/blob/master/README.md).
+
+## Table of Contents
+
+- [Installing GatorGrade](#installing-gatorgrade)
+  - [Including GatorGrade in Your Project](#including-gatorgrade-in-your-project)
+- [Using GatorGrade](#using-gatorgrade)
+  - [Running Checks](#running-checks)
+    - [Interpreting Output](#interpreting-output)
+  - [Generating a gatorgrade.yml File](#generating-a-gatorgrade.yml-file)
+    - [Configuring GatorGrade Checks](#configuring-gatorgrade-checks)
+- [Contributing to GatorGrade](#contributing-to-gatorgrade)
+  - [Installing Dev Environment](#installing-dev-environment)
+- [Contributors](#contributors)
+
+## Installing GatorGrade
+
+Installing GatorGrade requires a version of Python greater than 3.7. You can
+install GatorGrade from PyPi using `pip` or `pipx` package installers. This
+method of installation allows for GatorGrade to be accessible to all Python
+projects on your computer.
+
+### Including GatorGrade in Your Project
+
+To include GatorGrade in you project you can use `Poetry`. Once Poetry has been
+installed you can use `poetry add` to add the GatorGrade package.
```

Comment:
The steps for this can vary depending on how the user is setting up their virtual environment. The steps here make it sound like Poetry is required in order to use GatorGrade in the project. Maybe consider rephrasing a little bit.

Reply:
Maybe consider something along the lines of:

```
To include GatorGrade in you project we recommend you use the `Poetry` package manager. To install using `Poetry`  you can use `poetry add` to add the GatorGrade package. If you use another package manager, you can add GatorGrade as a dependency the same way you would with other `pip` or `pipx` packages!
```